account_move_fiscal_month/models/account_move.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class AccountMove(models.Model):
+
+    _inherit = 'account.move'
+
+    date_range_id = fields.Many2one(
+        comodel_name='date.range', string="Period",
+        compute='_compute_date_range', search='_search_date_range')
```

Comment:
I suggest naming this field `fiscal_month_id` or `date_range_fm_id`. That is more explicit, and leaves the door open to add a `fiscal_year_id` field.

Reply:
I updated the field name to ```date_range_fm_id```.